app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/MathExpressionInteractionsViewModel.kt

```diff
@@ -105,12 +105,17 @@ class MathExpressionInteractionsViewModel private constructor(
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
             pendingAnswerError,
-            answerText.isNotEmpty()
+            true
```

Comment:
There is no guarantee someone seeing this file has seen the others so the comments are still important.
```suggestion
       inputAnswerAvailable = true // Allow blank answer submission.
```

Reply:
right.